client/my-sites/domains/domain-management/edit/gdpr-consent-status-messages.js

```diff
@@ -0,0 +1,34 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { gdprConsentStatus } from 'lib/domains/constants';
+
+function getGdprConsentStatusMessage( domain ) {
+	switch ( domain.gdprConsentStatus ) {
+		case gdprConsentStatus.NONE:
+			return translate( 'Not available' );
+		case gdprConsentStatus.PENDING:
+			return translate( 'Pending registrar response' );
+		case gdprConsentStatus.PENDING_ASYNC:
+			return translate( 'Awaiting owner response' );
+		case gdprConsentStatus.ACCEPTED_CONTRACTUAL_MINIMUM:
+			return translate( 'Only for contractual needs' );
+		case gdprConsentStatus.ACCEPTED_FULL:
+			return translate( 'Full use of data allowed' );
+		case gdprConsentStatus.DENIED:
+			return translate( 'All use of non-contract data is denied' );
+		case gdprConsentStatus.FORCED_ALL_CONTRACTUAL:
+			return translate( 'Not required--only data for contract was collected' );
+		default:
+			return translate( 'Unavailable' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **6** times:
`translate( 'Site Unavailable' )` *ES Score: 18*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/show-gdpr-consent-status)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).